src/lib/AutocompleteTextInput.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import { FormElement } from "./index.js";
+  import { v4 as uuidv4 } from "uuid";
+
+  export let label;
+  export let name;
+  export let value;
+  export let options;
+  export let dataTestId = "";

```

Comment:
Need TS types, need to document what the options are (the order is arbitrary), don't need a name or that ID

```suggestion
  export let label: string;
  export let value: string;
  // A list of [value, label] pairs that can be autocompleted
  export let options: [string, string][];
```

Reply:
Reason I have name and label is because in the context of the choose page the label was big and long you might want something shorter for the name but maybe as with the ID stuff it being human readable might not be important.

`dataTestId` is used in choose your area on scheme sketcher, maybe there's a workaround though.

Reply:
I'd ignore generally everything about the autocomplete example from the choose area page; it's very old code